.github/workflows/build.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     - name: Run pre-commit
       uses: pre-commit/action@v2.0.0
     - name: Check for Sphinx doc warnings
+      if: contains(matrix.python-version, '3.8')
```

Comment:
👍

Reply:
@ravwojdyla thanks for your (offline) suggestion to do this. The workflow syntax works perfectly. But the build is failing on 3.8 too, so I need to look into that!